### PR TITLE
Connect Web Service with UI

### PR DESCRIPTION
**Pull request by jkusa**
# Description

Connect Web Service with UI

## Proposed Changes

- Change UI `rootURL` to `ui/`
- Fix topbar `rootURL` property
- Refactor sever generated config shape
- Refactor web service asset seving logic
- Fix datasourceName configuration in elide metadata query 
- Remove invalid addon config settings
- Run webservice tests in screwdriver

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by aklish — APPROVED** [review]

File: packages/app/mirage/config.js [hunk 1]
```diff
@@ -33,13 +33,15 @@ export default function() {
...
 
   // Mock bard facts + metadata
-  this.urlPrefix = `${config.navi.dataSources[0].uri}/v1`;
-  const metricBuilder = (metric, row, dimensionKey) => {
-    faker.seed(hashCode(`${row.dateTime}_${dimensionKey}_${metric}`));
-    return Number(faker.finance.amount());
-  };
-  BardLite.call(this, metricBuilder);
-  BardMeta.call(this);
+  config.navi.dataSources.forEach(({ uri }) => {
```
Comment (kevinhinterlong) [inline packages/app/mirage/config.js:36]: We could probably add an if to do this only `type === 'bard'`

File: packages/data/addon/services/navi-elide-apollo.ts [hunk 2]
```diff
@@ -72,9 +67,7 @@ export default class NaviElideApolloService extends ApolloService {
    */
   _buildURLPath(dataSourceName: string): string {
     const host = configHost({ dataSourceName });
-    const { namespace } = this;
-
-    return `${host}/${namespace}`;
+    return new URL(host, `${document.location.protocol}${document.location.host}`).href;
```
Comment (kevinhinterlong) [inline packages/data/addon/services/navi-elide-apollo.ts:70]: Is adding the base intentional? I don't think we expect people to pass `/graphql` as a host

**Review by kevinhinterlong — APPROVED** [review]

File: packages/app/mirage/config.js (hunk 1 above)
Comment (jkusa) [reply packages/app/mirage/config.js:36]: Agreed, we only have bard today in app. I'll make an issue.

Comment (jkusa) [reply packages/app/mirage/config.js:36]: https://github.com/yahoo/navi/issues/1111

File: packages/data/addon/services/navi-elide-apollo.ts (hunk 2 above)
Comment (jkusa) [reply packages/data/addon/services/navi-elide-apollo.ts:70]: In a single server scenario, this is what elide sends:
```js
{
   "user":"testuser",
   "dataSources":[
      {
         "name":"default",
         "uri":"/graphql/api/v1",
         "type":"elide",
         "timeout":900000
      }
   ],
   "appPersistence":{
      "name":"persistence",
      "uri":"/api/v1",
      "type":"elide",
      "timeout":900000
   },
   "FEATURES":{
      "enableDashboardsFilters":true,
      "enableTableEditing":true,
      "enableTotals":true
   }
}
```

Comment (jkusa) [reply packages/data/addon/services/navi-elide-apollo.ts:70]: via https://github.com/yahoo/navi/pull/1110/files#diff-530d4b5a68b71ffccd1308f7347bad17d76dfe298c400b866ab61a68775c63cbR20-R22

**Pull request merged**